deposit_contract.sol

```diff
@@ -26,13 +26,11 @@ interface IDepositContract {
 contract DepositContract is IDepositContract {
     uint constant GWEI = 1e9;
 
-    uint constant MIN_DEPOSIT_AMOUNT = 1000000000; // Gwei
     uint constant DEPOSIT_CONTRACT_TREE_DEPTH = 32;
     uint constant MAX_DEPOSIT_COUNT = 4294967295; // 2**DEPOSIT_CONTRACT_TREE_DEPTH - 1
```

Comment:
Why isn't this
```suggestion
    uint constant MAX_DEPOSIT_COUNT = 2**DEPOSIT_CONTRACT_TREE_DEPTH - 1;
```
?

Reply:
I would guess for no other reason than that Vyper cannot precompute it. I think your suggestion is an improvement

Reply:
One could also keep `deposit_count` as a `uint64`, and have this be `uint64(-1)`. `deposit_count` is casted to `uint64` wherever it appears now anyway